feat:[CDS-82193]: added notes for usergroup as expression in HA (#3918)
harness/developer-hub: docs/platform/approvals/adding-harness-approval-stages.md

  :::important
  Make sure no spaces are present in the expression values. In the above example, the format must be `_project_all_users,testUserGroup` to match the split criteria `,` .

  Make sure to use user group identifiers and not names in these expressions. 

>   Scope indicators needs to be prefixed for account and organization user groups 

Reviewer: ```suggestion
  You must prefix scope indicators for account and organization user groups, for example:
```
